historyserver/backend/collector/storage/interface.go

```diff
@@ -0,0 +1,14 @@
+package storage
+
+import "io"
+
+// StorageWriter is the interface for storage writer.
+type StorageWritter interface {
+	CreateDirectory(path string) error
```

Comment:
Can we rename `StorageWritter` to `StorageWriter`?
```suggestion
type StorageWriter interface {
	CreateDirectory(path string) error
```